platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/collections/LongListDisk.java

```diff
@@ -124,21 +256,61 @@ public synchronized boolean putIfEqual(final long index, final long oldValue, fi
      */
     @Override
     public void writeToFile(final Path newFile) throws IOException {
-        // finish writing to current file
-        fileChannel.force(true);
         // if new file is provided then copy to it
-        if (!file.equals(newFile)) {
-            try (final FileChannel fc =
-                    FileChannel.open(newFile, StandardOpenOption.CREATE, StandardOpenOption.WRITE)) {
-                fileChannel.position(0);
-                MerkleDbFileUtils.completelyTransferFrom(fc, fileChannel, 0, fileChannel.size());
-            }
+        try (final FileChannel fc = FileChannel.open(newFile, StandardOpenOption.CREATE, StandardOpenOption.WRITE)) {
```

Comment:
This method does not really override `AbstractLongList`. If `force(true)` invocation below is needed here, it's needed in the parent class as well.

Reply:
I'll remove the method and add ` fc.force(true)` to the parent method